tests/ui/invalid_base_class.stderr

```diff
@@ -0,0 +1,43 @@
+error[E0277]: pyclass `PyBool` cannot be subclassed
+ --> tests/ui/invalid_base_class.rs:4:19
+  |
+4 | #[pyclass(extends=PyBool)]
+  |                   ^^^^^^ required for `#[pyclass(extends=PyBool)]`
+  |
+  = help: the trait `PyClassBaseType` is not implemented for `PyBool`
+  = note: if you own `PyBool`, add `subclass` to the `#[pyclass]` macro: `#[pyclass(subclass)]`
+  = help: the following other types implement trait `PyClassBaseType`:
+            PyAny
+            PyArithmeticError
+            PyAssertionError
+            PyAttributeError
+            PyBaseException
+            PyBaseExceptionGroup
```

Comment:
The problem is that `BaseExceptionGroup` isn't defined until 3.11 (IIRC?).

You'll need to put a minimum Python version on this test, I think.

Reply:
Fixed that, thanks.